app/gui2/src/components/DocumentationPanel.vue

```diff
@@ -128,8 +127,14 @@ function handleBreadcrumbClick(index: number) {
       @forward="historyStack.forward()"
       @backward="historyStack.backward()"
     />
-    <DocsTags v-if="sections.tags.length > 0" class="tags" :tags="sections.tags" />
+    <DocsTags
+      v-if="sections.tags.length > 0"
+      class="tags"
+      :tags="sections.tags"
+      :groupColor="color"
```

Comment:
hmm... do we not have a CSS variable we can use to avoid having to pass this through? (this is a question, I'm not sure)

Reply:
We have, but to find the correct variable name, we need to parse the group name inside the tag, as we already do in the suggestion database. I find it more complicated than simply passing the correct color (as we do for breadcrumbs, anyway).